### PR TITLE
Minor UI fixes for invitations

### PR DESCRIPTION
**Pull request by kevinansfield**
issue #5652
- display "Sending Invite..." text after clicking "RESEND" link
- change "Invitation sent" alert to notification

### CODE REVIEW

File: core/client/app/components/gh-user-invited.js [hunk 1]
```diff
@@ -16,8 +17,10 @@ export default Ember.Component.extend({
     actions: {
         resend: function () {
             var user = this.get('user'),
-                notifications = this.get('notifications');
+                notifications = this.get('notifications'),
+                self = this;
```
Comment (halfdan) [inline core/client/app/components/gh-user-invited.js]: Not sure if this was discussed somewhere, but wouldn't it make sense to switch to arrow-functions here to make lines like the above obsolete?

Comment (kevinansfield) [inline core/client/app/components/gh-user-invited.js]: There's been quite a few places where I've been missing arrow functions. I remember some discussion about certain es6 features and our tooling not coping with it too well although I'm not sure if that was code climate, linting, or something else.

Would definitely be good to open a discussion on fat arrows and deciding whether we're standardizing on `export default Ember.Thing.extend({})` or `var Thing = {}; export default Thing;`.

Comment (ErisDS) [inline core/client/app/components/gh-user-invited.js]: RE export, we used to always do `var Thing = {}; export default Thing;` to keep it consistent with the node code, but recently there was a discussion in which it was determined that `export default Ember.Thing.extend({})` makes more sense in ES6 code, so the Ember code is moving towards that instead.

Comment (kevinansfield) [inline core/client/app/components/gh-user-invited.js]: I've already got a local commit that updates the majority of the exports on the ember side - I wanted something repetitive to do whilst I was mulling over a completely different issue! I can get that finished and pushed if we're heading in that direction.

Comment (ErisDS) [inline core/client/app/components/gh-user-invited.js]: :+1: :heart:  your mulling technique, I usually just stare out the window... :grin:

**Comment by JohnONolan** [comment]
:+1:

**Pull request merged**